### DISCUSSION TITLE
Ingame prices dollar pegged

### DISCUSSION THREAD

**Discussion by NoisyBucheron** [sunflower-land/sunflower-land#1241] [Ideas]
Hi, 

I was thinking of a way to get SFL token more valuable like bitcoin without a impact for new players.

What about to peg the value of ingame purchasses to dollar while still paying them in SFL?

Example: 
16 Jully > SFL Price 0.14$ USD
1 Wheat seed price ingame = 0.28SFL > 0.05$ USD IRL

If in 2 months SFL price is 1 dollar

16 September > SFL price 1$ USD
1 Wheat seed price ingame = 0.05SFL > 0.05$ USD IRL

- This system should be implemented with a refresh every 24H

- Halvings will cut the Dollar pegged price by half

- NFT's should not ask for SFL but ask for more ressources, this way more SFL would be burned and people would need to craft, trade and gather more ressources

Benefits:

- Make Halvings work
- Attract more players into the game with the SFL price rising
- Add value for the NFT's
- Can be applied to all ingame items

This is just a idea, not sure i'm right. I would like to hear some impressions.
What do you guys think?

**Comment by Yougottabekiddingme123** [comment]
Agree with this. Just to add  to this, I think SFL should have a max supply so that people value it and hold onto it as its value increases. Since the value of tools will be based on usd value, new players will not be negatively affected

**Comment by NotKeen** [comment]
My first thoughts: With this system you would have to peg both the inputs and the outputs to the dollar, which could be hard to sustain. Say you bought a Kuebicko **when SFL price was high** for 1000$ worth of SFL / inputs. And say you can increase your farm production by +$1 worth daily with it (all fantasy numbers).

Now what if we have a situation like now, where SFL price has dropped and demand has gone down a bit (bear market etc). These NFTs would lead to a lot of SFL printing (because of the $ peg), which could easily lead to a death spiral for SFL, as it is a feedback loop: Print SFL->Sell pressure decreases $ price->need to print even more SFL

Basically you get exactly the mess we have IRL ;-) 

imo.

**Comment by adamhannigan** [comment]
@NoisyBucheron Some great insights!

We did an exploration on using dynamic prices for everything in the game. The biggest challenge we found is that it killed the game progression system and what made the game 'fun'.

When designing prices, we calculate the required gameplay, effort and skill required to reach a certain milestone (i.e. planting wheat). When prices are fluctuating it makes balancing the challenge/reward system extremely difficult and as a side effect it either makes certain game aspects too hard or too easy for some.

The #1 factor that will affect SFL price is whether the game is fun and people are using the token.

I'm keen on exploring some dynamic prices for game items. For now we can't factor them into the resources that are tied to the core gameplay loop.

**Comment by clamcho** [comment]
what i think the price follow demand and supply not the token itself , maybe it will be more expensive trade p2p then you sell it on shop in your farm if demand > supply .
